taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs

```diff
@@ -215,13 +215,13 @@ private void CreateIfCondition(object sender, EventArgs e)
             var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginIfCommand).ToList();
 
             frmCommandEditor editor = new frmCommandEditor(automationCommands, null);
-            editor.selectedCommand = new BeginIfCommand();
+            editor.SelectedCommand = new BeginIfCommand();
             var res = editor.ShowDialog();
 
             if (res == DialogResult.OK)
             {
                 //get data
-                var configuredCommand = editor.selectedCommand as BeginIfCommand;
+                var configuredCommand = editor.SelectedCommand as BeginIfCommand;
                 var displayText = configuredCommand.GetDisplayValue();
                 var serializedData = Newtonsoft.Json.JsonConvert.SerializeObject(configuredCommand);
```

Comment:
Could Likely be reduced to JsonConvert.SerializeObject

Reply:
Same is the case here that it's just updating variable names/references in this Command File.